src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -31,6 +32,14 @@ class PeerActor(nodeInfo: NodeInfo) extends Actor with ActorLogging {
 
   val peerId = self.path.name
 
+  val waitForStatusInterval = 30.seconds
+  val waitForChainCheck = 15.seconds
+
+  //FIXME move this to props
+  lazy val DaoBlockNumber = 1920000
+  lazy val DaoBlockTotalDifficulty = BigInt("39490964433395682584")
```

Comment:
@LukasGasior1 I'm not sure. My intentention was to create a ForkValidator not specific for Dao but for any fork we want to check so thought putting this values within `application.conf` would make more sense